plugins/catalog-backend/src/service/standaloneServer.ts

```diff
@@ -47,7 +47,15 @@ export async function startStandaloneServer(
     const manager = DatabaseManager.fromConfig(
       new ConfigReader({
         backend: {
-          database: { client: 'better-sqlite3', connection: ':memory:' },
+          database: {
+            client: 'mysql2',
```

Comment:
I believe this was used for local testing. Should be reverted :)

Reply:
ah, yes, it was. Thanks for point it out. I'll change it.